src/main/java/hudson/plugin/versioncolumn/JVMConstants.java

```diff
@@ -27,5 +30,12 @@
         JDK_VERSION_NUMBER_TO_BYTECODE_LEVEL_MAPPING.put("8", JAVA_8);
         JDK_VERSION_NUMBER_TO_BYTECODE_LEVEL_MAPPING.put("1.9", JAVA_9);
         JDK_VERSION_NUMBER_TO_BYTECODE_LEVEL_MAPPING.put("9", JAVA_9);
+        JDK_VERSION_NUMBER_TO_BYTECODE_LEVEL_MAPPING.put("9.0", JAVA_9);
```

Comment:
Not sure what `.0` is expected to achiveve. Was it added to handle an/the actual issue?